sphynx/python/derive_html.py

```diff
@@ -0,0 +1,51 @@
+'''Run user code.'''
```

Comment:
Sorry, forgot to update this. The first half of this is just a copy of `derive.py`.